ghost/core/core/server/data/migrations/versions/5.55/2023-07-07-14-15-32-add-built-in-collections.js

```diff
@@ -0,0 +1,60 @@
+const logging = require('@tryghost/logging');
+const {default: ObjectID} = require('bson-objectid');
+const {createTransactionalMigration} = require('../../utils');
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        logging.info('Creating built in collections');
+
+        const existingIndexCollection = await knex('collections')
+            .where({
+                slug: 'index'
+            })
+            .first();
+
+        if (existingIndexCollection) {
+            logging.warn('Index collection already exists, skipping');
+        } else {
+            await knex('collections').insert({
+                id: (new ObjectID()).toHexString(),
+                name: 'Index',
```

Comment:
Detail to polish: we should decide if this built-in should be called "Latest" rather than Index. I think given the aim to make this easy to understand for users - it probably should be.

Reply:
Agreed, matching the product naming makes sense 👍 If we market it and use it as "Latest" in cards, it would be pretty confusing seeing "Index" in collections UI once we have one and figuring out what the hell this thing does 😄

Reply:
One thing that will be confusing though is "latest" implies there's ordering applied to the posts in the collection, which we are not forcing explicitly yet. I do think we should make the default order of posts in collections more explicit though.